src/DittoMachine.sol

```diff
@@ -290,16 +293,17 @@ contract DittoMachine is ERC721, ERC721TokenReceiver, ERC1155TokenReceiver, Clon
                 }
 
                 // reduce heat relative to amount of time elapsed by auction
-                if (cloneShape.term > block.timestamp) {
-                    uint256 termLength = BASE_TERM + TimeCurve.calc(heat);
-                    uint256 elapsed = block.timestamp - (cloneShape.term - termLength); // current time - time when the current term started
-                    // add 1 to current heat so heat is not stuck at low value with anything but extreme demand for a clone
-                    uint256 cool = (heat+1) * elapsed / termLength;
-                    heat = (cool > heat) ? 1 : Math.min(heat - cool + 1, type(uint8).max);
-                } else {
-                    heat = 1;
+                if (feeRefund == 0) { // if call is in same block as another keep the current heat
```

Comment:
`feeRefund` will be non-zero if the call is in the same block, right? This seems to be the opposite.